nss-ice-phantoms/BangazonAPI
# Jb change testing add include

# Description
Description of changes.
Made `include` into `_include` for customer url searching

Changed `PUT` and `DELETE` tests so that they are now more automated and don't require any hardcoding of id's... cause that was lame.

## Type of change

# Testing updated _include Instructions

1. `git fetch --all`
1. `git checkout jb-changeTestingAddInclude`
1. open in visual studio.
1. run bangazonAPI server.
1. `localhost:5000/api/customers?_include=payments` should show all customers with all their payment objects in the browser
1. `localhost:5000/api/customers?_include=products` should show all customers with all their product objects in the browser

# Testing new Tests Instructions
1. In visual studio open test explorer window
1. run all tests
1. assuming your bangazon database is populated all tests should pass


# Checklist:
